### PR TITLE
Wards skeleton

### PR DESCRIPTION
**Pull request by Nozziel**

### CODE REVIEW

File: tasks/districts/FetchMumbaiWardsTask.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+class FetchMumbaiWardsTask(luigi.Task):
+    def requires(self):
+        return DownloadFileTask(
+            file_url='http://stopcoronavirus.mcgm.gov.in/assets/docs/Dashboard.pdf')
+
+    def run(self):
+        df = scrape_mumbai_pdf(self.input().path)
+
+        df.to_csv(self.output().path, index=False)
+
```
Comment (rivera-lanasm) [inline tasks/districts/FetchMumbaiWardsTask.py:18]: just a note to myself for updating the phase 2 Ward PDF to Dropbox process: 
this is where you would suggest adding the method to write PDF to our drop box right?

and self.input().path points to the PDF file path on whatever machine the process is running on?

File: tasks/fetch_ward_data.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                ward_task = pd.read_csv(json_file)
+                ward_task = ward_task.set_index(['state', 'district', 'ward', 'date'])
+
+            # This needs to support overwriting existing data as well as adding new data
+            # TODO make a test for it
+            if all_wards is None:
+                all_wards = ward_task
+            else:
+                all_wards.concat(ward_task, axis=1)
+
```
Comment (rivera-lanasm) [inline tasks/fetch_ward_data.py:46]: reconsidering the placement of the method to write a given city's ward level data PDF to our drop box here instead, so as not to write a method for each city level task (like FetchMumbaiWardsTask()) we end up adding

File: tasks/districts/FetchMumbaiWardsTask.py (hunk 1 above)
Comment (Nozziel) [reply tasks/districts/FetchMumbaiWardsTask.py:21]: correct. I've updated the code to have that as a comment in there as well.

File: tasks/fetch_ward_data.py (hunk 2 above)
Comment (Nozziel) [reply tasks/fetch_ward_data.py:46]: actually, if all goes well Mumbai will be the only city where we need to do the saving.
The hope is that the other cities and maybe ven Mumbai itself will share the data online in google sheets, including historical data. So we would not need to story any backups ourselves.

File: backend/data/extract_mumbai_wards_pdf.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    return df
+
+
+def scrape_mumbai_pdf(source_file_path):
+    """
+    :param source_file_path: 
+    :remarks:
+    """
+    pdf = _read_pdf(source_file_path)
+    positive_cases_page = pdf.pages[20]  # TODO instead of hardcoded page, search for the correct page see notes above
```
Comment (rivera-lanasm) [inline backend/data/extract_mumbai_wards_pdf.py:54]: @Nozziel 

i could write a method to iterate through pdf.pages and return on page number where the "title region" (11.5, 10, 520, 78) contains "Ward-wise breakdown of positive cases", which might return 20 for example

Comment (Nozziel) [reply backend/data/extract_mumbai_wards_pdf.py:54]: Yeah sounds good. Create an issue for it :-)

**Pull request merged**